lib/replica_pools/connection_proxy.rb

```diff
@@ -33,7 +33,10 @@ def initialize(leader, pools)
       # this ivar is for ConnectionAdapter compatibility
       # some gems (e.g. newrelic_rpm) will actually use
       # instance_variable_get(:@config) to find it.
-      if ActiveRecord::VERSION::MAJOR >= 7
+
+      # 6.0 supports current.connection_config
+      # but warns of impending deprecation in 6.2
+      if ActiveRecord::VERSION::MAJOR >= 6
```

Comment:
Have we tested that `current.connection_db_config` works on 6.0?  I only ask because I found out yesterday that 6.0 doesn't have `ActiveRecord::DatabaseConfigurations::HashConfig.configuration_hash` even though 6.1 does (see https://github.com/kickstarter/replica_pools/pull/40#issuecomment-1505819615).